vars/fcosKolaTestIso.groovy

```diff
@@ -0,0 +1,46 @@
+// Run kola testiso
+// Available parameters:
+//     cosaDir:        string  -- cosa working directory
+//     extraArgs:      string  -- extra arguments to pass to `kola testiso`
+//     extraArgs4k:    string  -- extra arguments to pass to 4k `kola testiso`
+//     scenarios:      string  -- scenarios to pass to `kola testiso`
+//     scenarios4k:    string  -- scenarios to pass to `kola testiso`
+//     skipMetal4k:    boolean -- skip metal4k image
+def call(params = [:]) {
+    def cosaDir = utils.getCosaDir(params)
+    def extraArgs = params.get('extraArgs', "");
+    def extraArgs4k = params.get('extraArgs4k', "");
+    def scenarios = params.get('scenarios', "");
+    def scenarios4k = params.get('scenarios4k', "");
+
+    testIsoRuns = [:]
+    testIsoRuns["metal"] = {
+        try {
+            if (params['scenarios']) {
+                shwrap("cd ${cosaDir}  && kola testiso -S ${extraArgs} --scenarios ${scenarios} --output-dir tmp/kola-testiso-metal")
+            } else {
+                shwrap("cd ${cosaDir}  && kola testiso -S ${extraArgs} --output-dir tmp/kola-testiso-metal")
+            }
```

Comment:
I think we could simplify this by just have `--scenarios` be part of the `scenarios` variable if it's non-zero, but we can do that as a follow-up!